README.md

```diff
@@ -1,28 +1,31 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/VMClarity-logo-dark-bg-horizontal@4x.png">
-  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/VMClarity-logo-light-bg-horizontal@4x.png">
-  <img alt="VMClarity Logo" src="./assets/logos/VMClarity-logo-light-bg-horizontal@4x.png">
+  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/OpenClarity-logo-dark-bg.png">
+  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/OpenClarity-logo-light-bg.png">
+  <img alt="OpenClarity Logo" src="./assets/logos/OpenClarity-logo-light-bg.png">
 </picture>
 
 [![Slack Invite](https://img.shields.io/badge/Slack-Join-blue?logo=slack)](https://outshift.slack.com/messages/vmclarity)
-[![Go Reference](https://pkg.go.dev/badge/github.com/openclarity/vmclarity.svg)](https://pkg.go.dev/github.com/openclarity/vmclarity)
-![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/openclarity/vmclarity/main-merge.yml?style=flat-square&branch=main)
-[![Go Report Card](https://goreportcard.com/badge/github.com/openclarity/vmclarity)](https://goreportcard.com/report/github.com/openclarity/vmclarity)
 
-VMClarity is an open source tool for agentless detection and management of Virtual Machine
+<!--[![Go Reference](https://pkg.go.dev/badge/github.com/openclarity/vmclarity.svg)](https://pkg.go.dev/github.com/openclarity/vmclarity)-->
+
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/openclarity/openclarity/main-merge.yml?style=flat-square&branch=main)
+
+<!--[![Go Report Card](https://goreportcard.com/badge/github.com/openclarity/vmclarity)](https://goreportcard.com/report/github.com/openclarity/vmclarity)-->
+
+OpenClarity is an open source tool for agentless detection and management of Virtual Machine
 Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.
 
-<img src="./assets/vmclarity_demo.gif" alt="VMClarity demo" />
+<img src="./assets/OpenClarity-demo.gif" alt="OpenClarity demo" />
 
-Join [VMClarity's Slack channel](https://outshift.slack.com/messages/vmclarity) to hear about the latest announcements and upcoming activities. We would love to get your feedback!
+Join [OpenClarity's Slack channel](https://outshift.slack.com/messages/vmclarity) to hear about the latest announcements and upcoming activities. We would love to get your feedback!
```

Comment:
The slack channel should be renamed to `openclarity` or a new channel should be created?